tests/test_roundtrip.rs

```diff
@@ -45,87 +45,90 @@ fn test_roundtrip() {
         vec
     };
 
-    let path = format!("{}/tests/data/spidermonkey/ecma_2/**/*.js", env!("CARGO_MANIFEST_DIR"));
-    debug!(target: "test_roundtrip", "Starting test_roundtrip from {}", path);
-
-    for entry in glob::glob(&path)
-        .expect("Invalid glob pattern")
-    {
-        let entry = entry.expect("Invalid entry");
-
-        // Parse and preprocess file.
-
-        println!("Parsing {:?}.", entry);
-        let mut ast    = parser.parse_file(entry.clone())
-            .expect("Could not parse source");
-        debug!(target: "test_roundtrip", "Source: {}", ast.pretty(2));
-        debug!(target: "test_roundtrip", "Annotating {:?}.", entry);
-
-        let mut ast = binjs::specialized::es6::ast::Script::import(&ast)
-            .expect("Could not import AST");
-        binjs::specialized::es6::scopes::AnnotationVisitor::new()
-            .annotate_script(&mut ast);
-
+    let test_files = vec!["tests/data/spidermonkey/ecma_2/**/*.js", "tests/data/facebook/single/*.js"];
+    
+    for test_file in test_files {
+        let path = format!("{}/{}", env!("CARGO_MANIFEST_DIR"), test_file);
+        debug!(target: test_file, "Starting {} from {}", test_file, path);
```

Comment:
Oh, I hadn't seen that you had put `test_file` as `target`. Could you restore the debug messages that were there previously?

Reply:
Poor me, I am on it.